File: src/Practica2/Cliente.java
package Practica2;

import java.util.ArrayList;
import java.util.Date;

public class Cliente {
    /*
    Cread los siguientes atributos
     * nombre: Nombre del cliente.
     * apellidos: Apellidos del cliente.
     * dni: Número del DNI.
     * email: Dirección email.
     * fecha_nacimiento: La fecha de nacimiento del cliente.
     * libros: Un array de libros comprados por ese cliente.
     */
    private String nombre;
    private String apellidos;
    private int dni;
    private String email;
    private Date fecha_nacimiento;
    private ArrayList<Libro>listaLibros;
    //Método constructor
    public Cliente (String nombre,String apellidos, int dni,String email, Date fecha_nacimiento, ArrayList<Libro> listaLibros){
        this.nombre=nombre;
        this.apellidos=apellidos;
        this.dni=dni;
        this.email=email;
        this.fecha_nacimiento=fecha_nacimiento;
        this.listaLibros=listaLibros;
    }

    // Getters y setters
    public String getNombre(){
        return this.nombre;
    }
    public void setNombre(String pNombre){
        this.nombre=pNombre;
    }
    public String getApellidos(){
        return this.apellidos;
    }
    public void setApellidos(String pApellidos){
        this.apellidos=pApellidos;
    }
    public int getDni(){
        return this.dni;
    }
    public void setDni(int pDni){
        this.dni=pDni;
    }
    public String getEmail(){
        return this.email;
    }
    public void setEmail(String pEmail){
        this.email=pEmail;
    }
    public Date getFecha_nacimiento(){
        return this.fecha_nacimiento;
    }
    public void setFecha_nacimiento(Date pFecha_nacimiento){
        this.fecha_nacimiento=pFecha_nacimiento;
    }
    public ArrayList<Libro> getListaLibros(){
        return this.listaLibros;
    }
    public void setListaLibros(ArrayList<Libro> pListaLibros){
        this.listaLibros=pListaLibros;
    }
    
}

File: bin/Practica2Python/Cliente.py
class Cliente:
    # Atributos
    nombre=""
    apellidos=""
    dni=0
    email=""
    fecha_nacimiento=""
    listaLibros=[]
    #Método constructor
    def __init__(self,nombre,apellidos,dni,email,fecha_nacimiento,listaLibros):
        self.nombre=nombre
        self.apellidos=apellidos
        self.dni=dni
        self.email=email
        self.fecha_nacimiento=fecha_nacimiento
        self.listaLibros=listaLibros
    # Getters y setters
    def getNombre(self):
        return self.nombre
    def setNombre(self,pNombre):
        self.nombre=pNombre
    def getApellidos(self):
        return self.apellidos
    def setApellidos(self,pApellidos):
        self.apellidos=pApellidos
    def getDni(self):
        return self.dni
    def setDni(self,pDni).
        self.dni=pDni
    def getEmail(self):
        return self.email
    def setEmail(self,pEmail):
        self.email=pEmail
    def getFecha_nacimiento(self):
        return self.fecha_nacimiento
    def setFecha_nacimiento(self,pFecha_nacimiento).
        self.fecha_nacimiento=pFecha_nacimiento
    def getListaLibros(self):
        return self.listaLibros
    def setListaLibros(self,pListaLibros)
        self.listaLibros=pListaLibros